### DISCUSSION TITLE
Language/Locale Label Missing

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#13578] [Archive]
Hi,

I went into the *_translations table and put those 2 attributes inside a detail group **metadata** like this:

![image](https://user-images.githubusercontent.com/7398499/170527825-7f465da6-a97a-4545-ac30-b168abb229a2.png)

Because of the detail group, the locale labels against the fields are missing on the content entry form. Does this look like a bug?

![image](https://user-images.githubusercontent.com/7398499/170527956-c20aa650-5942-4df8-8392-0bd9af2fe3e0.png)

thanks
  Matt

**Comment by rijkvanzanten** [comment]
> Does this look like a bug?

It does to me!

**Reply by st11x** [reply]
@rijkvanzanten  should I convert this to an issue, or let you do it? Thanks.

**Reply by rijkvanzanten** [reply]
I'll do it 👍🏻
